data/json/recipes/ammo/pistol.json

```diff
@@ -287,6 +287,42 @@
     "proficiencies": [ { "proficiency": "prof_handloading" } ],
     "components": [ [ [ "500_casing", 1 ] ], [ [ "lgrifle_primer", 1 ] ], [ [ "chem_black_powder", 19 ] ], [ [ "copper", 5 ] ] ]
   },
+  {
+    "result": "bp_50ae_jhp",
+    "type": "recipe",
+    "activity_level": "LIGHT_EXERCISE",
+    "category": "CC_AMMO",
+    "subcategory": "CSC_AMMO_PISTOL",
+    "skill_used": "fabrication",
+    "difficulty": 9,
```

Comment:
Is there a reason why this is so difficult? .50 AE is a straightwall cartridge, reloading it should be pretty similar to other straight cartridges.